File: create_facility.php
<?php
	session_start();
    $facility_name = $_POST["facility_name"];
	$address = $_POST["address"];
	$city = $_POST["city"];
	$state = $_POST["state"];
	$zip = $_POST["zip"];
	$poc = $_POST["poc"];
	$userid = $_SESSION['userid'];

	$facility = "'".$facility_name."' '".$address."' '".$city."' '".$state."' '".$zip."' '".$poc."' '".$user_id."'";
	$execStr = 'python /var/www/html/reunite/scripts/create_facility.py '.$facility;
	$result = exec($execStr);
	//echo $execStr;
	//echo $result;
	//echo "created facility: ".$facility_name;
	header("Location: display_facilities.php");
?>

File: scripts/create_facility.py
import sys
import facilities

facility_name = sys.argv[1]
address = sys.argv[2]
city = sys.argv[3]
state = sys.argv[4]
zip = sys.argv[5]
poc = sys.argv[6]
user_id = sys.argv[7]

facilities.create_facility(facility_name, address, city, state, zip, poc, user_id)

